<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>481</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Titr</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>اطلاعات</string>
     </attribute>
     <widget class="QLabel" name="piclabel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>111</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>عکس</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>نام</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>فامیل</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>شماره موبایل</string>
      </property>
     </widget>
     <widget class="QLabel" name="firstlabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lastlabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="phonelabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>رزومه</string>
      </property>
     </widget>
     <widget class="QLabel" name="resumelabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>109</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>تعداد مراجعات</string>
      </property>
     </widget>
     <widget class="QLabel" name="numlabel">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>641</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column>
       <property name="text">
        <string>تاریخ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ساعت</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>دکتر</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>وضعیت</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>360</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>بروز رسانی</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>نوبت گیری</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
selection-background-color: rgb(64, 64, 64);
selection-color: rgb(0, 255, 0);
alternate-background-color: rgb(128, 128,128);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Time_Box">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Doc_Box">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="appo_Button">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>360</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>B Titr</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ثبت نوبت</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 127);</string>
      </property>
     </widget>
     <widget class="QLabel" name="Check_Label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>279</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>مراجعات</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>391</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>629</width>
         <height>389</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>پیام ها</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPatient_Profile">
    <property name="font">
     <font>
      <family>B Nazanin</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>پرتال بیمار</string>
    </property>
   </widget>
   <addaction name="menuPatient_Profile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
